discovery-provider/docker-compose.dev.yml

```diff
@@ -5,7 +5,7 @@ services:
   celery-worker:
     build: .
     restart: always
-    command: sh -c '/wait && exec celery -A src.worker.celery worker'
+    command: sh -c '/wait && exec celery -A src.worker.celery worker --loglevel info'
```

Comment:
is this not configurable through the loglevel_celery config? if not - we might want to remove that since it's misleading. @dmanjunath wy think

Reply:
yeah we should comment out that line maybe put a comment in the config file explaining where you can set the variable

Reply:
@hareeshnagaraj yeah i tried not adding this command line flag for the log level and using what we have configured in our own logger. for some reason, celery will recognize the formatting we set, but not the log level. instead, celery will use its default log level, which is also set at WARNING. with trial and error, using this command line flag was the only way to get it set to whatever level desired

i'll leave a comment in the config about this behavior :)